content/support.rst

```diff
@@ -0,0 +1,212 @@
+
+A separate knowledge base is currently under evaluation.
+
+
+Chat
+====
+
+`Olark <https://www.olark.com>`_ is currently used for chat.
+
+Each chat transcript is forwarded to the Support email list and subsequently
+creates a new ticket in the helpdesk.
+
+
+Licenses and support contract
+=============================
+
+See the `Support options page <https://www.sftpplus.com/support/options.html>`_
+which describes what options are available.
+If a support request goes beyond the standard option (such as a new
+product feature), the Sales team will determine the quote.
+
+See the `Life cycle page <https://www.sftpplus.com/product/life-cycle.html>`_ 
+for details about the product life cycle.
+
+If a feature request is obtained, check that it is included in the
+`Product roadmap <https://www.sftpplus.com/product/roadmap.html>`_ 
+
+License information is available from Sales team to check if a customer
+has an existing Support license.
+
+
+Support Request Management
+==========================
+
+
+Logging a support request
+-------------------------
+
+Support requests can be received either through the online support form,
+direct email or phone call.
+
+Email and support forms are the preferred methods.
+This is due to the possibility of miscommunication that can happen in a
+phone call.
+
+Support requests can be logged by partners, customers or the Support team.
+
+When a support request is logged by a customer or partner, an email is sent to
+the support team.
+
+When one of the members of the support team logs a support request, he/she
+should consider obtaining and collating as much information as possible,
+ensuring that any of the support team members can pick up the SR at any time
+without recourse to the owner.
+
+All files related to a support request should also be available in the SR
+system.
+
+
+Initial Assessment
+------------------
+
+When a new support request is logged it should receive a response within the
+same or next business day.
+
+The SR will be allocated to an Owner, who should do the initial assessment:
+
+* Verify all details logged against the call, checking what additional
+  information is required;
+
+* Make initial contact with the originator, acknowledge that we have receive
+  the request;
+
+* Verify understanding of initial information provided;
+
+* Ask for additional information if required and/or provided initial diagnosis;
+
+* If more time is needed before providing an answer, inform about the
+  approximate time.
+
+If a customer asks for multiple support questions in a single email, it is
+better to respond in multiple emails, creating a new ticket for each email.
+
+
+Managing a Support Request 
+--------------------------
+
+Once a ticket has been responded, close the ticket (a notification will not be
+sent to the customer).
+
+The ticket will re-open when the customer responds.
+
+Do not close the ticket if the response is a 'holding' type of response.
+
+
+Escalating a Support Request
```

Comment:
The details can be deleted so that it only talks about what to do when a defect is found.